R/scale-.r

```diff
@@ -45,7 +45,9 @@
 #'   scales, which ways use [scales::rescale()].
 #' @param oob Function that handles limits outside of the scale limits
 #'   (out of bounds). The default ([scales::censor()]) replaces out of
-#'   bounds values with `NA`.
+#'   bounds values with `NA`. ([scales::squish()]) can also be used,
```

Comment:
I think this might now be better as a bulleted list. Do you want to have a go at formatting it that way?

Reply:
Sure, on it.